snuba/datasets/factory.py

```diff
@@ -43,6 +45,7 @@ def get_dataset(name: str) -> Dataset:
 
     dataset_factories: MutableMapping[str, Callable[[], Dataset]] = {
         "events": EventsDataset,
+        "events_new": ErrorsDataset,
```

Comment:
Why is this `events_new` if it is for the `ErrorsDataset`?

Reply:
Because Sentry depends on "events" dataset in most of the product. The rollout will be in two phases. First replace Events with this dataset while still calling it events (thus I call it events_new and then plan to swap it and remove events_new). Then we can, if we want, create an alias named errors and make the change in sentry, thus removing the events at all.

Reply:
I personally would avoid `new` and instead use something like `v2`. If the transition takes a while, the `new` won't be so new.

Reply:
I think it would be confusing to have discoverV2 and eventsV2 together while there is an actual Sentry feature called events-v2 ?
Will find a better name

Reply:
The sentry eventsv2 feature is the same as discoverv2 sadly. Names are forever hard, but once discover1 is gone, I plan on renaming all the things.